store/sqlstore/upgrade.go

```diff
@@ -672,12 +671,12 @@ func UpgradeDatabaseToVersion511(sqlStore SqlStore) {
 		// Enforce all teams have an InviteID set
 		var teams []*model.Team
 		if _, err := sqlStore.GetReplica().Select(&teams, "SELECT * FROM Teams WHERE InviteId = ''"); err != nil {
-			mlog.Error("Error fetching Teams without InviteID: " + err.Error())
+			mlog.Error("Error fetching Teams without InviteID:", mlog.Err(err))
```

Comment:
What about adding a `team` or `team_id` parameter to this?  Also we can remove the `:`.

Reply:
done